arangod/Aql/QueryRegistry.h

```diff
@@ -170,6 +181,11 @@ class QueryRegistry {
     bool _finished = false;
 
     cluster::CallbackGuard _rebootTrackerCallbackGuard;
+
+    Scheduler::WorkHandle _destructionTrackingTask;
+    /// @brief serves to log the information if the query destruction took too
+    /// long
+    std::shared_ptr<QueryDestructionContext> _queryDestrcutionContext;
```

Comment:
```suggestion
    std::shared_ptr<QueryDestructionContext> _queryDestructionContext;
```
also all usages, of course :)